frontend/src/components/WofEditPopup.vue

```diff
@@ -0,0 +1,143 @@
+<!-- 
+Component changes something.
+Example:
+<wof-edit-popup title="Edit username" v-if="this.isOpen" :isOpen="this.isOpen" currentValueName="Username" currentValue="wofUser123" @close="changeState" @edit="consoleLog" />
+
+Simple methods to test the component:
+  data(){
+    return{
+      isOpen: true
+    }
+  },
+  methods: {
+    changeState(){
+      if(this.isOpen==true){
+                this.isOpen=false;
+            }
+            else{
+                this.isOpen=true;
+            }
+    },
+    consoleLog(newValue, newValue2){
+      console.log(newValue+": "+newValue2);
+    }
+  }
+-->
+
+
+<template>
+<div class="wof-edit-popup">
```

Comment:
WofInfoBox should wrap everything. This div with "wof-edit-popup" should be inside this like that:
```
<wof-info-box>
    <div class="wof-edit-popup">
            and put here body and loading
    </div>
</wof-info-box>
```